platformio_tubes.ini

```diff
@@ -164,3 +164,40 @@ lib_ignore = ${env:esp32-c3-athom.lib_ignore}
   ${tubes_no_mic.lib_ignore}
 lib_deps =
   ${tubes_no_mic.lib_deps}
+
+# ------------------------------------------------------------------------------
+# ESP32 Athom
+# ------------------------------------------------------------------------------
```

Comment:
Can you add a comment with the specific device?

Want to make sure we don't confuse people who have the C3 (which is also Athom)